suite-common/wallet-types/src/transaction.ts

```diff
@@ -9,7 +9,7 @@ import {
     CardanoInput,
     CardanoOutput,
 } from '@trezor/connect';
-import { Network } from '@suite-common/wallet-config';
+import { Network, NetworkSymbol } from '@suite-common/wallet-config';
```

Comment:
When you have added _NetworkSymbol_ to imports, fix it everywhere in this file instead of _Network['symbol']_.

Reply:
https://github.com/trezor/trezor-suite/pull/6140/commits/5a38c9aea55e9811f7fecdf7679ca1a86081f8ec